frontend/src/scenes/session-recordings/player/playerMetaLogic.ts

```diff
@@ -58,6 +58,13 @@ export const playerMetaLogic = kea<playerMetaLogicType>({
                     height: snapshot.data['height'],
                 }
             },
+            {
+                resultEqualityCheck: (prev, next) => {
+                    // Only update if the resolution values have changed (not the object reference)
+                    // stops PlayerMeta from re-rendering on every player position
+                    return objectsEqual(prev, next)
```

Comment:
The player meta rendered on every tick because this selector was comparing object instances to determine change and not the width and height they carry